views/clucks/index.ejs

```diff
@@ -0,0 +1,55 @@
+<%- include('../partials/header') %>
+
+<div class="clucks-list">
+
+  <% for (let cluck of clucks) {
+
+<!-- Calculate time -->
+
+function timeSince(date) {
+
+var seconds = Math.floor((new Date() - date) / 1000);
+
+var interval = Math.floor(seconds / 31536000);
```

Comment:
Note there are some magic numbers happening in this part of the code. May want to clarify them as you did previously.